src/oapv.c

```diff
@@ -2153,5 +2153,26 @@ int oapvd_info(void *au, int au_size, oapv_au_info_t *aui)
 #endif // ENABLE_DECODER
 ///////////////////////////////////////////////////////////////////////////////
 
+#if defined(OAPV_VERSION_HEADER)
+
+char * oapv_version() { 
+    static char oapv_ver[64];
+    snprintf(oapv_ver, sizeof(oapv_ver), "%d.%d.%d", OAPV_VERSION_MAJOR, OAPV_VERSION_MINOR, OAPV_VERSION_PATCH);
+    return oapv_ver; 
+}
+
+char * oapv_libversion() { 
+    static char oapv_libver[64];
+    snprintf(oapv_libver, sizeof(oapv_libver), "%d.%d.%d", OAPV_LIB_VERSION_MAJOR, OAPV_LIB_VERSION_MINOR, OAPV_LIB_VERSION_PATCH);
+    return oapv_libver; 
+}
+
+#else
+
 static char *oapv_ver = "0.1.13.1";
+static char *oapv_libver = "1.0.0";
+
 char * oapv_version() { return oapv_ver; }
+char * oapv_libversion() { return oapv_libver; }
```

Comment:
I think this would be better as

```
unsigned oapv_libversion(void) {
    return (OAPV_LIB_VERSION_MAJOR << 16) |
           (OAPV_LIB_VERSION_MINOR <<  8) |
            OAPV_LIB_VERSION_PATCH;
}
```

And then adding the following to oapv_version.h

```
#define OAPV_LIB_MAJOR(v) (((v) >> 16) & 0xFF)
#define OAPV_LIB_MINOR(v) (((v) >>  8) & 0xFF)
#define OAPV_LIB_PATCH(v) ( (v)        & 0xFF)
```

This is a lot more library user friendly than a string for runtime version checks.
oapv_version() can remain a string just fine, but it should be const char*.